x-pack/plugins/threat_intelligence/public/containers/integrations_guard/integrations_guard.tsx

```diff
@@ -7,26 +7,75 @@
 
 import { EuiLoadingLogo } from '@elastic/eui';
 import React, { FC } from 'react';
+import { HttpSetup } from '@kbn/core/public';
+import { useRequest, UseRequestConfig } from '@kbn/es-ui-shared-plugin/public';
 import { EmptyPage } from '../../modules/empty_page';
 import { useIndicatorsTotalCount } from '../../modules/indicators';
 import { SecuritySolutionPluginTemplateWrapper } from '../security_solution_plugin_template_wrapper';
 
+type IntegrationInstallStatus = 'installed' | 'installing' | 'install_failed';
+
+let httpClient: HttpSetup;
+
+const PACKAGES_CONFIG: UseRequestConfig = {
```

Comment:
i know that the endpoint is `packages` but we refer to them as to integrations, should we call it smth like `integrationsRequestConfig`? btw why the upper-case?

Reply:
I wanted to stick to the Fleet naming convention as I felt it would be easier to look at their code and ping point things there. I can change it though no problem

Reply:
updated everything to use the word `integration`